lib/deprecation-cop-status-bar-view.coffee

```diff
@@ -22,6 +22,13 @@ class DeprecationCopStatusBarView extends View
     @subscriptions.add atom.packages.onDidUnloadPackage @updateDeprecatedSelectorCount
     @subscriptions.add atom.packages.onDidActivatePackage @updateDeprecatedSelectorCount
 
+    @subscriptions.add atom.keymaps.onDidReloadKeymap (event) =>
+      @updateDeprecatedSelectorCount() if event.path is atom.keymaps.getUserKeymapPath()
+
+    try
+      userStylesheetFile = new File(atom.styles.getUserStyleSheetPath())
```

Comment:
> Just want to throw this out there, instead of using File here you might be able to do this via the atom.styles API.

Great idea! :bulb: :zap: Using that as of fff645b47e7eb7dca6aacb244180a08097cdac30, which also includes the other changes you suggested.

And thanks for removing those empty `catch` blocks in 618eabe9b106118b07894f34dae49de0a3330c8e -- totally didn't know that was possible.